### PR TITLE
Link current contents

### PR DESCRIPTION
**Pull request by ben181231**
connect #13

### CODE REVIEW

File: tools/lib/routes-loader.js [hunk 1]
```diff
@@ -16,6 +18,79 @@ export default function(source) {
...
+          line += `}),`;
+        } else {
+          line += `'${perRouteName}': () => Promise.resolve(`;
+          line += `  ${factory.name}({`;
+          line += `    name: "${perRouteData.name}",`;
+          line += `    title: "${perRouteData.title}",`;
+          line += `    ${contentParam}`;
+          line += `    ${menuParam}`;
+          line += `  })`;
+          line += `),`;
```
Comment (rickmak) [inline tools/lib/routes-loader.js]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

You can save the `line +=`

**Comment by rickmak** [comment]
I think we want to remove the hugo heading in this PR?

I am referring following
```
+++ date = "2015-09-24T18:33:04+08:00" draft = false title = "Container, Databases and Records"

+++
```

File: tools/lib/routes-loader.js (hunk 1 above)
Comment (ben181231) [inline tools/lib/routes-loader.js]: @rickmak Okay.

**Comment by ben181231** [comment]
@rickmak Okay. I will remove the hugo headings after rebasing this branch.

**Comment by ben181231** [comment]
@rickmak PR updated.

**Pull request merged**